lib/src/kvs/ds.rs

```diff
@@ -821,25 +847,132 @@ impl Datastore {
 	// It is handy for testing, because it allows you to specify the timestamp,
 	// without depending on a system clock.
 	pub async fn tick_at(&self, ts: u64) -> Result<(), Error> {
-		self.save_timestamp_for_versionstamp(ts).await?;
+		let vs = self.save_timestamp_for_versionstamp(ts).await?;
+		if FFLAGS.change_feed_live_queries.enabled() {
+			self.process_lq_notifications(ts, vs).await?;
+		}
 		self.garbage_collect_stale_change_feeds(ts).await?;
 		// TODO Add LQ GC
 		// TODO Add Node GC?
 		Ok(())
 	}
 
 	// save_timestamp_for_versionstamp saves the current timestamp for the each database's current versionstamp.
-	pub async fn save_timestamp_for_versionstamp(&self, ts: u64) -> Result<(), Error> {
+	pub async fn save_timestamp_for_versionstamp(
```

Comment:
Signature change here. Should we perhaps make this `pub(crate)`? Can also have a method stub for the accidentally exposed pub signature